site/content/docs/developer-guide/plugin-manifest.md

```diff
@@ -99,6 +100,9 @@ Krew makes it possible to install the same plugin on different operating systems
 (e.g., `windows`, `darwin` (macOS), and `linux`) and different architectures
 (e.g., `amd64`, `386`, `arm`, `arm64` and `ppc64le`).
 
+See [all supported platforms](https://github.com/kubernetes-sigs/krew/releases).
+Only plugins in platforms that krew itself is distributed (Release Assets) are supported.
```

Comment:
also wanted to float the idea of having a static list here. i know its not great to have to update in multiple places when something new is supported but this isnt a frequent occurrence. when ppc64le was added the contributor also [remembered to update the docs](https://github.com/kubernetes-sigs/krew/pull/755/files#diff-bbfd929e7884a7e220b4bcddf4c4f992cde11c8b31fab08f874f7caf9ec87fa1R100). an option to help any drift between what's supported and the docs would be to add a comment (maybe to [here](https://github.com/kubernetes-sigs/krew/blob/master/hack/krew.yaml#L28)) as a reminder to update the docs area with the new platform.

if not then i think it reads better as part of the previous paragraph like
```suggestion
The supported platforms for plugins are the ones that Krew are distributed in.
See all supported platforms on the [releases page](https://github.com/kubernetes-sigs/krew/releases).
```

Reply:
Got it, updated.
PTAL thanks.